csharp-basics/exercises/Arithmetic/CozaLozaWoza/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CozaLozaWoza
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            int j = 1;
+            for (int i = 1; i <= 10; i++)
+            {
+                for (; j <= 11*i; j++)
```

Comment:
Kas tas pa vienu ;?

Reply:
Es izveidoju j mainīgo ārpus for loop, lai varētu izlaist tā resetu pēc katras iterācijas, līdz ar to parādās vientuļš ;